pkg/dyninst/dyninstexprlang/dyninstexprlang.go

```diff
@@ -0,0 +1,126 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+//go:build linux_bpf
+
+// Package dyninstexprlang provides utilities for working with expressions in the expression language
```

Comment:
what about `exprlang` for this package name? We're already underneath dyninst

Reply:
I prepended `dyninst` to try to distinguish that this is for the expression language specific to DI/LD. Expression is pretty overloaded. But I agree it's too wordy/redundant. Maybe `exprlang`  is the best it's gonna get.